Trustroots/trustroots
# Member search: split into separate reusable components

#### Proposed Changes

* Split member search component into smaller, more generic, easier to re-use components:
  - Add `LoadingIndicator` component
  - Add `NoContent` component
  - I want to eventually merge [`<ContactPresentational />`](https://github.com/Trustroots/trustroots/blob/master/modules/contacts/client/components/ContactPresentational.js) with `<UserSummary />`
* Use CSS grid instead of Bootstrap columns and manually splitting arrays into two
* Simplified some components while at it as well.

#### Testing Instructions

* Go to `http://localhost:3000/search/members` and test that results work (try e.g. with your logged-in username) and that "no results" work as well. Try with data that give 3+ results.
    <img width="1189" alt="Screenshot 2020-01-19 at 21 48 42" src="https://user-images.githubusercontent.com/87168/72687342-9215d100-3b05-11ea-9fd9-7f65b0516ea4.png">

* Go to profile's contacts list tab (add some contacts first) — for the search bar to appear you need to have a bunch) and confirm that list and filtering works
    <img width="881" alt="Screenshot 2020-01-19 at 21 47 04" src="https://user-images.githubusercontent.com/87168/72687323-6bf03100-3b05-11ea-8492-6a3632e43a74.png">

* No visual changes nowhere
